src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -953,7 +953,9 @@ class TezosDatabaseOperationsTest
           medium = mu,
           high = mu + sigma,
           timestamp = latest,
-          kind = ops.head.kind
+          kind = ops.head.kind,
+          cycle = None,
+          level = Some(0)
```

Comment:
Are we testing here that actual latest value is the one used in the average result?

Reply:
no, we're not, those values were generated, I'm not why they are not `Nones`, as they are optional in the table